googletrans/client.py

```diff
@@ -29,20 +30,19 @@ class Translator(object):
     :param user_agent: the User-Agent header to send when making requests.
     :type user_agent: :class:`str`
 
-    :param proxies: proxies configuration. 
-                    Dictionary mapping protocol or protocol and host to the URL of the proxy 
-                    For example ``{'http': 'foo.bar:3128', 'http://host.name': 'foo.bar:4012'}``
+    :param raise_exception: if `True` then raise exception if smth will go wrong
+    :type raise_exception: boolean
     """
 
-    def __init__(self, service_urls=None, user_agent=DEFAULT_USER_AGENT, proxies=None):
```

Comment:
Could you explain me why you removed proxy support instead of just adding an needed argument? I think you did by mistake.

Reply:
Yeah, it's my mistake - no purpose here